docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -279,7 +284,7 @@ Mapping character types
 ^^^^^^^^^^^^^^^^^^^^^^^
 
 Trino's ``VARCHAR(n)`` maps to ``VARCHAR2(n CHAR)`` if ``n`` is no greater than
-4000. A larger or unbounded ``VARCHAR`` maps to ``NCLOB``.
+1.    A larger or unbounded ``VARCHAR`` maps to ``NCLOB``.
```

Comment:
do not change.

Reply:
Whoop, not sure how that happened. Chalk it up to an errant vscode keystroke.

Reply:
It's back again. I think we should reformat the line so that the `N.` isn't at the beginning of the line. I guess something is treating it as a list.

Reply:
Agreed & fixed.